### ISSUE TITLE
Getting a lot of "Parsing error: Unexpected token"

### ISSUE THREAD

**Issue by joshxyzhimself** [microsoft/vscode#106190]
Version: 1.48.2
Commit: a0479759d6e9ea56afa657e454193f72aef85bd0
Date: 2020-08-25T10:56:10.074Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.4.0-37-generic snap

When I'm working with a large file, linter just gets stuck with that.

On ESLint OUTPUT tab I'm getting a lot of 

```sh
[Info  - 6:35:49 pm] Cannot read property 'value' of null Occurred while linting /home/user/repo/Profile.js:97
[Info  - 6:35:49 pm] Cannot read property 'value' of null Occurred while linting /home/user/repo/Profile.js:97
[Info  - 6:35:50 pm] Cannot read property 'value' of null Occurred while linting /home/user/repo/Profile.js:97
[Info  - 6:35:50 pm] Cannot read property 'value' of null Occurred while linting /home/user/repo/Profile.js:97
[Info  - 6:35:51 pm] Cannot read property 'value' of null Occurred while linting /home/user/repo/Profile.js:97
[Info  - 6:35:51 pm] Cannot read property 'value' of null Occurred while linting /home/user/repo/Profile.js:97
```

Which is simply a code block of 

```js
    const [profile] = await postgres`
      SELECT *
      FROM "profiles"
      WHERE id = ${profile_id}; // line 97
    `;
```

---

```js
    "eslint": "^7.5.0",
    "eslint-config-airbnb": "^18.2.0",
    "eslint-loader": "^4.0.2",
    "eslint-plugin-import": "^2.21.2",
    "eslint-plugin-jsx-a11y": "^6.3.1",
    "eslint-plugin-react": "^7.20.3",
    "eslint-plugin-react-hooks": "^4.0.8",
```

```js
module.exports = {
  globals: {
    window: true,
    document: true,
  },
  parser: 'babel-eslint',
  parserOptions: {
    ecmaVersion: 6,
    sourceType: 'module',
  },
  env: {
    browser: true,
    node: true,
  },
  extends: ['airbnb', 'airbnb/hooks'],
  plugins: ['jsx-a11y', 'react'],
};

```

**Comment by joshxyzhimself** [comment]
Solved at https://github.com/eslint/eslint/issues/13662#issuecomment-687760548

TLDR: was using `babel-eslint` instead of [@babel/eslint-parser](https://github.com/babel/babel/tree/main/eslint/babel-eslint-parser)

Thanks to @mdjermanovic :100:
